src/objects/Term_Grade__c.object

```diff
@@ -0,0 +1,233 @@
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The class in which the student is received this grade.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The class in which the student is received this grade.</inlineHelpText>
+        <label>Course Connection</label>
+        <referenceTo>Course_Enrollment__c</referenceTo>
+        <relationshipLabel>Term Grades</relationshipLabel>
+        <relationshipName>Term_Grades</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Earned_Credits__c</fullName>
+        <description>The number of credits earned for this course and grading period.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The number of credits earned for this course and grading period.</inlineHelpText>
+        <label>Earned Credits</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>2</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Letter_Grade__c</fullName>
+        <description>The letter grade the student earned for this course. For example, &quot;A+&quot; or any other text value.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The letter grade the student earned for this course. For example, &quot;A+&quot; or any other text value.</inlineHelpText>
+        <label>Letter Grade</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Numerical_Grade__c</fullName>
+        <description>The numerical grade the student earned for this course.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The numerical grade the student earned for this course.</inlineHelpText>
+        <label>Numerical Grade</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>5</scale>
+        <trackTrending>false</trackTrending>
+        <type>Number</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Percent_Grade__c</fullName>
+        <description>The grade the student earned for this course represented as a percentage. For example, 87.5%, which is equivalent to a B letter grade in most cases.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The grade the student earned for this course represented as a percentage. For example, 87.5%, which is equivalent to a B letter grade in most cases.</inlineHelpText>
+        <label>Percent Grade</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>0</scale>
```

Comment:
Is this correct? I may need to deploy to my org to see. This reads to me that only full percentages are allowed.

Reply:
def. not correct. Should be digits to the right of the decimal. I believe we settled on 3 for % and 5 for numerical grade.

Reply:
Confirmed that I was not able to save with decimal points after the %

Reply:
@xinnanlisf This field needs to be adjusted to match the spec definition.